lyncs_io/utils.py

```diff
@@ -44,3 +44,11 @@ def default_names(i=0):
     "Infinite generator of default names ('arrN') for entries of an archive."
     yield f"arr{i}"
     yield from default_names(i + 1)
+
+
+def prod(val):
```

Comment:
Please use the function from `lyncs_utils`
```
from lyncs_utils import prod
```
Usually I put there generic utils, you can look in a future if there is what you need or make it :)

Reply:
Oh even better. I will update it to that then.